tests/test_static_files.py

```diff
@@ -0,0 +1,53 @@
+import os.path
+
+from playwright.async_api import async_playwright
+
+from lona.static_files import StyleSheet, Script
+from lona.html import Widget
+from lona import LonaView
+
+
+def setup_app(app):
+    @app.route('/')
+    class MyLonaView(LonaView):
+        def handle_request(self, request):
+            return 'SUCCESS'
+
+
+async def test_static_files(lona_app_context):
+    some_existing_file = os.path.basename(__file__)
+
+    # isn't discovered in setup_app()
```

Comment:
What does this mean?

Reply:
Would it be more clear if I change comment to this?
```py
# if you put widget into setup_app(), lona will not discover static files
```

Reply:
Oh! That sounds like a bug. We should add a "FIXME: " then

Reply:
I've fixed comment.

I'm not sure it's a bug. However need to investigate for sure.
I'll create an issue for that.

Reply:
Let's add the fixme anyway. The widget should be part of the setup stage, not the test i think

Reply:
Added